src/features/user/TreeMapper/MySpecies/MySpeciesForm.tsx

```diff
@@ -81,21 +74,20 @@ export default function MySpeciesForm() {
 
   const addSpecies = async (species: NewSpecies) => {
     setIsUploadingData(true);
-    const data = {
+    const data: SpeciesPayload = {
       aliases:
-        species.aliases || species.aliases !== ''
+        (species.aliases || species.aliases !== ''
           ? species.aliases
-          : species.scientificSpecies?.name,
-      scientificSpecies: species.scientificSpecies?.id,
+          : species.scientificSpecies?.name) ?? null,
+      scientificSpecies: species.scientificSpecies?.id ?? null,
     };
     try {
-      await postAuthenticatedRequest({
-        tenant: tenantConfig.id,
-        url: `/treemapper/species`,
-        data,
-        token,
-        logoutUser,
-      });
+      await postApiAuthenticated<Species, SpeciesPayload>(
+        `/treemapper/species`,
+        {
+          payload: data,
+        }
```

Comment:
Can this be named better? If we're calling it data, we may as well call it payload.